commit/plugin_e2e_test.go

```diff
@@ -40,53 +46,53 @@ import (
 )
 
 const (
-	destChain    = ccipocr3.ChainSelector(1)
-	sourceChain1 = ccipocr3.ChainSelector(2)
-	sourceChain2 = ccipocr3.ChainSelector(3)
+	destChain         = ccipocr3.ChainSelector(1)
+	sourceChain1      = ccipocr3.ChainSelector(2)
+	sourceChain2      = ccipocr3.ChainSelector(3)
+	ArbAddr           = ocr2types.Account("0xa100000000000000000000000000000000000000")
+	ArbAggregatorAddr = ocr2types.Account("0xa2000000000000000000000000000000000000000")
+
+	EthAddr           = ocr2types.Account("0xe100000000000000000000000000000000000000")
+	EthAggregatorAddr = ocr2types.Account("0xe200000000000000000000000000000000000000")
 )
 
-func TestPlugin_E2E_AllNodesAgree(t *testing.T) {
-	ctx := tests.Context(t)
-	lggr := logger.Test(t)
+var (
+	oracleIDs = []commontypes.OracleID{1, 2, 3}
+	peerIDs   = []libocrtypes.PeerID{{1}, {2}, {3}}
 
-	donID := uint32(1)
-	oracleIDs := []commontypes.OracleID{1, 2, 3}
-	peerIDs := []libocrtypes.PeerID{{1}, {2}, {3}}
-	require.Equal(t, len(oracleIDs), len(peerIDs))
+	ArbPrice = big.NewInt(1).Mul(big.NewInt(5), big.NewInt(1e18))
+	EthPrice = big.NewInt(1).Mul(big.NewInt(7), big.NewInt(1e18))
```

Comment:
Just left a comment about this [in another PR](https://github.com/smartcontractkit/chainlink-ccip/pull/217#discussion_r1801148135). I like that you used 1 instead of 0, but I think this is a little better
```suggestion
	ArbPrice = new(big.Int).Mul(big.NewInt(5), big.NewInt(1e18))
	EthPrice = new(big.Int).Mul(big.NewInt(7), big.NewInt(1e18))
```

Reply:
agreed.